### PR TITLE
enh: sparkle-ization of app design

### PR DESCRIPTION
**Pull request by spolu**
- Use Tab instead of subNavigation + simple title so that it can be closed
- Rollout buttons and icons
- Removes the old Button components

cc @Duncid I'm not super fan of the title + tab below but it's not that bad. Given that it's not our core focus I presume it's good enough but let me know if you have some ideas on that.

Screenshot from 2023-08-31 18-33-48

### CODE REVIEW

File: front/components/providers/AI21Setup.tsx [hunk 1]
```diff
@@ -67,7 +80,7 @@ export default function AI21Setup({ owner, open, setOpen, config, enabled }) {
 
   return (
     <Transition.Root show={open} as={Fragment}>
-      <Dialog as="div" className="relative z-10" onClose={() => setOpen(false)}>
+      <Dialog as="div" className="relative z-30" onClose={() => setOpen(false)}>
```
Comment (fontanierh) [inline front/components/providers/AI21Setup.tsx:83]: FMI, why do we need to move to `z-30` everywhere ?

File: front/components/sparkle/AppLayout.tsx [hunk 2]
```diff
@@ -290,7 +290,7 @@ export default function AppLayout({
           </div>
           <div
             className={classNames(
-              "fixed left-0 right-0 top-0 z-30 flex h-16 flex-row lg:left-80",
+              "fixed left-0 left-12 right-0 top-0 z-30 flex h-16 flex-row lg:left-80",
```
Comment (fontanierh) [inline front/components/sparkle/AppLayout.tsx:293]: Maybe we want to remove the `left-0` then no ? Because I imagine only one of those 2 can apply ?

**Review by fontanierh — DISMISSED** [review]
LGTM but will test quickly

File: front/components/providers/AI21Setup.tsx (hunk 1 above)
Comment (spolu) [reply front/components/providers/AI21Setup.tsx:83]: Standardizing all Dialogs as some of them woudl conflict with the AppLayout title :+1:

File: front/components/sparkle/AppLayout.tsx (hunk 2 above)
Comment (spolu) [reply front/components/sparkle/AppLayout.tsx:293]: Yep yep

**Comment by fontanierh** [comment]
Screenshot 2023-09-01 at 14 02 03


Super nit, but is the disabled state of the `deploy` button as expected ?

**Comment by spolu** [comment]
Yes => cc @Duncid

**Comment by spolu** [comment]
(@Duncid already noted that it's a bit meh meh)

**Comment by spolu** [comment]
Fixes #1220

**Pull request merged**